voicevox_engine/user_dict/model.py

```diff
@@ -90,10 +90,12 @@ def check_is_katakana(cls, pronunciation: str) -> str:
     def check_mora_count_and_accent_type(self) -> Self:
         if self.mora_count is None:
             rule_others = (
-                "[イ][ェ]|[ヴ][ャュョ]|[トド][ゥ]|[テデ][ィャュョ]|[デ][ェ]|[クグ][ヮ]"
+                "[イ][ェ]|[ヴ][ャュョ]|[ウクグトド][ゥ]|[テデ][ィェャュョ]|[クグ][ヮ]"
             )
-            rule_line_i = "[キシチニヒミリギジビピ][ェャュョ]"
-            rule_line_u = "[ツフヴ][ァ]|[ウスツフヴズ][ィ]|[ウツフヴ][ェォ]"
+            rule_line_i = (
+                "[キシチニヒミリギジヂビピ][ェャュョ]|[キスニヒミリギズビピ][ィ]"
+            )
```

Comment:
ここのルールなかなか分類が難しいですね！！
ここのスとズは`rule_line_u`とかぶってるかも･･･？（自信ちょっとなし）

```suggestion
            rule_line_i = (
                "[キシチニヒミリギジヂビピ][ェャュョ]|[キニヒミリギビピ][ィ]"
            )
```

Reply:
気づいていませんでした。
ありがとうございます！
エディターの方のPRも合わせて修正しました。
https://github.com/VOICEVOX/voicevox/pull/2293/commits/56c015f3e5cdb2c42b5bc342be2f9188125d82b3